src/engraving/types/symnames.cpp

```diff
@@ -3805,7 +3805,7 @@ constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_
     "Punctum auctum, ascending",
     "Punctum auctum, descending",
     "Augmentum (mora)",
-    QT_TRANSLATE_NOOP("symUserNames", "Caesura"),
+    QT_TRANSLATE_NOOP("symUserNames", "Chant caesura"),
```

Comment:
This is not the name used in SMuFL!
Yes, ithe glyph is `chantCaesure`, but the clear text name is "Caesura" (unfortunatly, complain to SMuFL).
Whenever ...tools/fonttools/smufl2sym.[{sh,bat} is run again, it'd revert this change.

Reply:
Esp. the commented out code in the mtests checking for duplicates, as in 3.x this was only in the master palette